index.html

```diff
@@ -607,9 +607,19 @@ <h3>Profile Resolution Semantics</h3>
       <h2>Constraints</h2>
       <section id="Document Encoding">
         <h3>Document Encoding</h3>
-        <p>
-          A Document Instance MUST use UTF-8 character encoding as specified in [[UNICODE]].
-        </p>
+        <p>A <a>Document Instance</a> MUST be concretely encoded as a well-formed XML 1.0 [[!xml]] document using the UTF-8
+          character encoding as specified in [[UNICODE]].</p>
+    
+          <p>The resulting [[!xml]] document MUST NOT contain any of the following physical structures:</p>
+    
+          <ul>
+            <li><em>entity declarations</em>; and</li>
+    
+            <li><em>entity references</em> other than to <em>predefined entities</em>.</li>
+          </ul>
+    
+          <p class="note">The resulting [[xml]] document can contain <em>character references</em>, and <em>entity references</em> to
```

Comment:
Can we add a link to https://www.w3.org/TR/xml/#dt-charref for "character references"? and maybe give an example? Also what's the use case for allowing character references when the encoding is UTF-8?

Reply:
Allowing character references is the default, I think, so I'd look for a use case for disallowing them instead - I can't think of one. In particular, allowing character references can help to unambiguously specify a character, or to know what a character is in case the font being used to review the XML file does not support the character.

Reply:
Links added in ff6d63b.

Thinking more about the idea of prohibiting character references, I think it would be an unwelcome surprise to find that they are not supported, to many. Also, I realise I don't understand why you're linking the use of character references to the UTF-8 encoding, particularly?

Reply:
Let's say if we were to allow ASCII as a possible encoding, to encode any character you'd have to use character references. With UTF-8 as the encoding you can represent any Unicode code point, thus there is no need for character reference. Not having to support character references would make processors simpler but I won't argue more. We can get feedback from implementers.